packages/e2e.test.foundry-sdk-generator/generateMockOntology.js

```diff
@@ -14,21 +14,29 @@
  * limitations under the License.
  */
 
+// @ts-check
+import { __testSeamOnly_NotSemverStable__GeneratePackageCommand as GeneratePackageCommand } from "@osdk/foundry-sdk-generator";
 import { apiServer } from "@osdk/shared.test";
-import { rm } from "fs/promises";
-import { join } from "path";
-import { GeneratePackageCommand } from "../generate/index.js";
+import * as fs from "node:fs/promises";
+import * as path from "node:path";
+import { fileURLToPath } from "node:url";
 
-const dir = `${__dirname}/../generatedNoCheck/`;
-export async function setup() {
+const dir = path.join(
```

Comment:
ESM doesn't have `__dirname`. This worked in vitest 1 because in reality vitest was converting all code to CJS before executing it and CJS has `__dirname`. However the tests are actually being run as ESM now and thus this breaks.